app/controllers/dogs_controller.rb

```diff
@@ -0,0 +1,31 @@
+class DogsController < ApplicationController
+
+  def index
```

Comment:
For testing this controller, would something like:

```
RSpec.describe "Test", type: :request do
  describe "request list of all tests" do
    user = User.create(email: 'test@user.com', name: 'Test User')
    get dogs_path
    expect(response).to be_successful
    expect(response.body).to include("Thinking about fostering")
  end
end
```

Reply:
(We don't have to go too nuts with this right now since we don't have the models to back this up yet; figured we can figure that out this coming Monday).

Reply:
Yep, can do. Though I think we are using minitest?

Reply:
Test added!